examples/hex_neighbors_radius_chunks.rs

```diff
@@ -541,18 +541,18 @@ fn main() {
                 })
                 .set(ImagePlugin::default_nearest()),
         )
-        .add_plugin(TilemapPlugin)
+        .add_plugins(TilemapPlugin)
         .init_resource::<CursorPos>()
         .init_resource::<HighlightRadius>()
         .init_resource::<TileHandleHexCol>()
         .init_resource::<TileHandleHexRow>()
         .init_resource::<FontHandle>()
-        .add_startup_systems((spawn_chunks, apply_system_buffers).chain().in_set(SpawnChunksSet))
-        .add_startup_system(spawn_tile_labels.after(SpawnChunksSet))
-        .add_systems((camera_movement, update_cursor_pos).chain().in_base_set(CoreSet::First))
-        .add_system(swap_map_type)
-        .add_system(hover_highlight_tile_label.after(swap_map_type))
-        .add_system(update_radius.after(hover_highlight_tile_label))
-        .add_system(highlight_neighbor_labels.after(update_radius))
+        .add_systems(Startup, (spawn_chunks, apply_deferred).chain().in_set(SpawnChunksSet))
+        .add_systems(Startup, spawn_tile_labels.after(SpawnChunksSet))
+        .add_systems(First, (camera_movement, update_cursor_pos).chain())
+        .add_systems(Update, swap_map_type)
+        .add_systems(Update, hover_highlight_tile_label.after(swap_map_type))
+        .add_systems(Update, update_radius.after(hover_highlight_tile_label))
+        .add_systems(Update, highlight_neighbor_labels.after(update_radius))
```

Comment:
I haven't tested this, but I believe that these `add_systems` calls can be combined as well. You should be able to combine the Startup systems in one call, and the Update systems in another call I believe.

Reply:
A great idea! I'm not going to hold the PR up for it though. :) Feel free to open another PR and I'll merge it for sure. 👍